src/Adapter/Debug/DebugModeConfiguration.php

```diff
@@ -153,4 +156,26 @@ private function updateDebugMode($enableStatus)
             return (true === $enableStatus) ? $this->debugMode->enable() : $this->debugMode->disable();
         }
     }
+
+    // @todo: should be extracted
+    private function persistDebugIps($debugIps)
+    {
+        if (!empty($debugIps)) {
+            $ips = explode(',', $debugIps);
+
+            $code = '<?php return ' . var_export($ips, true) . ';';
+        } else {
+            $code = '<?php return [];';
+        }
+
+        file_put_contents(__DIR__.'/../../../config/debug_ip.php', $code);
```

Comment:
This could be improved I guess.

Reply:
Better to generate a yaml or json file instead of php